src/efi.rs

```diff
@@ -24,15 +23,9 @@ use crate::util::CommandRunExt;
 /// The path to the ESP mount
 pub(crate) const MOUNT_PATH: &str = "boot/efi";
 
-#[derive(Serialize, Deserialize)]
+#[derive(Default)]
 pub(crate) struct EFI {}
 
-impl EFI {
-    pub(crate) fn new() -> Self {
```

Comment:
What about repurposing this as a `pub(crate) fn new_component() -> Box<dyn Component>`?

Reply:
If I understand correctly, this doesn't seem much clearer to me than inlining the `Box::new(..:default())` - I reworked to do that.  Does that work?